shop/src/main/java/ro/msg/learning/shop/entities/Stock.java

```diff
@@ -0,0 +1,22 @@
+package ro.msg.learning.shop.entities;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity @Data
+public class Stock implements Serializable {
+    @EmbeddedId StockId stockId;
```

Comment:
Please look at the review for OrderDetails

Reply:
I suggested to use composite primary keys. So no changes needed.